.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
+        our contributing guidelines for more details on how we assess a brand's
+        popularity.
```

Comment:
This should probably include mention of (a) what to do if the alexa rank is too low and (b) what to do if a non-web brand is requested. Any suggestions?

Reply:
Could say "If the Alexa rank is too low or the brand does not have a website, please include any relevant information to help us assess the brand's popularity."

Reply:
Hmm :thinking: First, I think we should ask for either Alexa rank or GitHub stars here (potentially more if we have more standard metrics). Second, I think that's heading in the right direction. What about: 

---

Provide either [an Alexa rank]() or the number of GitHub stars.

If these are not available or not sufficient you can still open the issue. Please provide any information regarding the brand's popularity you think is relevant in the description below.

---

Reply:
That is much better, though what should the field be called now? `Popularity Metric` perhaps?

Reply:
Sounds good :+1: